bundles/org.openhab.core/src/main/resources/OH-INF/i18n/i18n.properties

```diff
@@ -0,0 +1,40 @@
+config.system.i18n.language.label = Language
+config.system.i18n.language.description = <![CDATA[
+                <p>The default language that should be used. If not specified, the system default locale is used.</p>
+                <p>The ISO 639 alpha-2 or alpha-3 language code (if there is no alpha-2 one).</p>
```

Comment:
Just a thought - as the valid options are provided and the UI displays them, I wonder if the "ISO code" indication and the examples are necessary in these parameters' descriptions.
For those configuring them in the services.cfg file, these hints could be provided as comments.

Reply:
Yes, I agree. The examples are necessary for textual configuration but not in the UI. We can remove it here and the other configuration parameters for this service.